### ISSUE TITLE
link to other resources

### ISSUE THREAD

**Issue by arademaker** [ekaf/ski#2]
many ontologies have links to different versions of WN. For instance, SUMO (https://github.com/ontologyportal/sumo/tree/master/WordNetMappings). Does it make sense to generalize the idea from this repo making it a general repository for linking WN to other resources?

**Comment by ekaf** [comment]
Yes, this project's name is "Sense Key Index for inter-operability between WordNet-related projects": the idea is that any WN-related resource can be linked to any other WN version through the SKI. This repo includes examples of combining this approach with ILI identifiers, and it would certainly make sense to also include SUMO.

**Comment by arademaker** [comment]
Not only SUMO but many other ontologies could be linked to WN. BFO is one additional example...

**Comment by ekaf** [comment]
There is an important distinction between databases that already have WordNet identifiers assigned to their entities, like SUMO, OMW, FrameNet, VerbNet (and many others...), and those like BFO that may be linked transitively through an already linked resource.

Any resource that is linked to any WordNet version can use the SKI directly, to get corresponding synsets from any other resource that uses any other WordNet version.

But resources that don't yet have a direct mapping to WordNet cannot use the SKI directly. Still, the SKI may be useful for research and development towards creating the new mappings.
